scipy/stats/_qmc.py

```diff
@@ -1125,6 +1127,7 @@ class LatinHypercube(QMCEngine):
     --------
     Generate samples from a Latin hypercube generator.
 
+    >>> import numpy as np
```

Comment:
```suggestion
```
I don't think this is needed here because `np` is not used anywhere in this block of the example. Actually, it's not used in _any_ block of code in the example, so it should not be placed in a later block, either.